Clases Entidad

MagaPache opened the pull request:
Se agregan las clases entidad y los esqueletos de clases anexas para completar durante el desarrollo :)

MagaPache committed:
Clases Entidad

Se agregan las clases entidad y los esqueletos de clases anexas para completar durante el desarrollo :)

fernandodelima commented on Project/Desktop/src/Model/Curso.java:
```diff
@@ -0,0 +1,112 @@
+    private double costo;
+    private Timestamp diaHorario;
+    private int cargaHoraria;
+
```
Falta campo cupo y aula.

fernandodelima replied:
Corregido

fernandodelima commented on Project/Desktop/src/Model/DatosGenerales.java:
```diff
@@ -0,0 +1,101 @@
+    private Date fechaNacimiento;
+    private String telefono;
+    private String email;
+
```
Cambiar nombre de atributos para hacerlos generales nombre y apellido.

MagaPache replied:
Listo ;)

fernandodelima commented on Project/Desktop/src/Model/Inscripcion.java:
```diff
@@ -0,0 +1,81 @@
+    private int idInscripcion;
+    private Curso curso;
+    private Cursante cursante;
+    private boolean estado;
```
Revisar el tipo de dato, debe ser int, no boolean.

MagaPache committed:
Update Datos Generales

Se realizan las correcciones ;)

fernandodelima committed:
Update Curso

Ok !

MagaPache committed:
Update Inscripcion

Se realizan los cambios solicitados para el tipo de dato de las variables estado y aprobado

MagaPache commented:
Gracias sensei :)

fernandodelima committed:
Update Inscripcion.java
